web/src/engine/osk/src/views/oskView.ts

```diff
@@ -20,7 +20,7 @@ import {
 } from 'keyman/engine/keyboard';
 import { createUnselectableElement, getAbsoluteX, getAbsoluteY, StylesheetManager } from 'keyman/engine/dom-utils';
 import { EventListener, LegacyEventEmitter } from 'keyman/engine/events';
-import { type MutableSystemStore, type SystemStoreMutationHandler } from 'keyman/engine/js-processor';
+import { type MutableSystemStore, type SystemStoreMutationHandler } from 'keyman/engine/keyboard';
```

Comment:
We could add this to the import of the the other types of keyman/engine/keyboard in lines 14-19.